sw/device/silicon_creator/rom_ext/BUILD

```
@@ -252,6 +249,7 @@ opentitan_binary(
         "//sw/device/silicon_creator/lib:manifest_def",
         "//sw/device/silicon_creator/lib/ownership:test_owner",
         "//sw/device/silicon_creator/lib/ownership/keys/fake",
+        IMM_ROM_EXT_SECTIONS["a"]["main"],
```

Can you just use the dep directly?
```suggestion
       //sw/device/silicon_creator/imm_rom_ext:main_section_slot_a
```

Same below.

---

Thanks, this approach looks much clearer!